### ISSUE TITLE
New summon mechanic on 6.2 PTR

### ISSUE THREAD

**Issue by Sharparam** [SharpWoW/LibBodyguard-1.0#2]
Currently on live, the bodyguard casts a "Reputation Check" spell when being summoned, on the PTR this has changed, as pointed out by Blazeflack on [this ticket on WoWAce](http://www.wowace.com/addons/libbodyguard-1-0/tickets/1-ptr-6-2-bodyguard-frame-will-not-show-on-initial-pickup/).

It is now the player casting two spells when summoning a bodyguard: `Maintain <bodyguard name>` and `Summon <bodyguard name>`. Below are details about the UNIT_SPELLCAST_SUCCEEDED events:

```
UNIT_SPELLCAST_SUCCEEDED
arg1: "player"
arg2: "Maintain Delvar Ironfist"
arg3: ""
arg4: 0
arg5: 173638
```

```
UNIT_SPELLCAST_SUCCEEDED
arg1: "player"
arg2: "Summon Delvar Ironfist"
arg3: ""
arg4: 0
arg5: 173633
```

These will need to be incorporated into the library to properly detect the bodyguard summon again. The relevant spell IDs should be able to be found through Wowhead.

**Comment by Sharparam** [comment]
Should be fixed in 67611202fb9e931c0c56304b78713d8bccb6d30d. Awaiting confirmation.

**Comment by Sharparam** [comment]
Confirmed fixed on WoWAce, closing.
